dietpi/dietpi-software

```diff
@@ -12370,6 +12427,26 @@ _EOF_
 
 			fi
 			ln -s /mnt/dietpi_userdata/steam /root/.steam
+			if [[ $G_HW_ARCH == 2 ]]; then
+				# Stop Steam from trying to install i386 packages on ARM
+				G_EXEC rm /etc/apt/sources.list.d/steam.list 
+				cat << '_EOF_' > /usr/bin/steamdeps
+#!/bin/bash
+
+exit 0
+_EOF_
+				# Add Steam desktop shortcut
+				cat << '_EOF_' > /home/dietpi/Desktop/steam.desktop
+[Desktop Entry]
+Name=Steam
+Exec=sh -c "STEAMOS=1 STEAM_RUNTiME=1 /usr/bin/steam +open steam://open/minigameslist %U"
```

Comment:
To avoid a continuously active shell process, we could use:
```
dash -c "STEAMOS=1 STEAM_RUNTiME=1 exec /usr/bin/steam +open steam://open/minigameslist %U"
```
or:
```
env STEAMOS=1 STEAM_RUNTiME=1 /usr/bin/steam +open steam://open/minigameslist %U
```
- `exec` replaces the wrapping shell with the called command, so the shell process will be gone.
- `env` is explicitly used to call commands with environment variables and itself does not stay as active parent process, so that one looks like a cleaner solution.